src/core/prism.ts

```diff
@@ -162,7 +162,13 @@ export class Prism {
 	 * Usually a language definition like `Prism.languages.markup`.
 	 * @returns The highlighted HTML.
 	 * @example
-	 * Prism.highlight('var foo = true;', 'javascript');
+	 * import { Prism } from './src/core/prism';
```

Comment:
I might miss some essential things, but those imports should be shorter if I could say so. Like, `import Prism, { Token } from "prismjs"`, `import javascript from "prismjs/languages"`. Is this something we should fix in `package.json`? Do I understand correctly that we use these examples to show how to use Prism as an NPM package?

Reply:
On a higher leve, these are meant to be small snippets, not full code examples. I think it’s ok if this doesn’t include the import code, which is different depending on tooling/env.